docs/production/deploying-to-cloud/azure/setup.mdx

```diff
@@ -220,7 +220,7 @@ resource "azurerm_container_group" "container_group" {
         You can run the following command to see all the resources that will be created
         by Terraform: `terraform plan`
 
-        By default, here are the **[resources](resources)** that will be
+        By default, here are the **[resources](production/deploying-to-cloud/azure/resources)** that will be
```

Comment:
Did you test this url? This looks a relative url. I'm not sure whether it works.

Reply:
Nice catch! Fixed